controllers/admin/AdminCustomerThreadsController.php

```diff
@@ -1171,9 +1171,7 @@ public function syncImap()
                         }
                     }
                 }
-                if ($fetch_succeed) {
-                    Db::getInstance()->execute('INSERT INTO `' . _DB_PREFIX_ . 'customer_message_sync_imap` (`md5_header`) VALUES (\'' . pSQL($md5) . '\')');
-                }
+                Db::getInstance()->execute('INSERT INTO `' . _DB_PREFIX_ . 'customer_message_sync_imap` (`md5_header`) VALUES (\'' . pSQL($md5) . '\')');
```

Comment:
`fetch_succeed` can be false because of line 1167 :thinking:

Reply:
No because at the line 1169, there is a continue.

Reply:
Yes but only if an exception is raised. :thinking:

Reply:
But at the next step, `fetch_succeed` will be define to `true`.

Reply:
Oh, I get it!